.vib/mongodb-sharded/vib-publish.json

```diff
@@ -36,6 +36,18 @@
           "params": {
             "endpoint": "lb-mongodb-sharded-mongodb"
           }
+        },
+        {
+          "action_id": "goss",
+          "params": {
+            "resources": {
+              "path": "/.vib/mongodb-sharded/goss"
+            },
+            "remote": {
+              "workload": "sts-mongodb-sharded-mongos"
+            },
+            "vars_file": "vars.yaml"
+          }
```

Comment:
You only included the GOSS action in the `vib-publish` workflow but forgot about `vib-verify`.

Reply:
Damn, I totally missed this. Added the action and increased the timeout to some of the tests; of the 4 pipelines, 1 of them did timeout. Ref: [action run(with goss)](https://github.com/FraPazGal/charts/actions/runs/3089823682)